hw/bsp/imxrt/family.c

```diff
@@ -91,13 +104,8 @@ static void init_usb_phy(uint8_t usb_id) {
   usb_phy->TX = phytx;
 }
 
-void board_init(void)
-{
-  // make sure the dcache is on.
-#if defined(__DCACHE_PRESENT) && __DCACHE_PRESENT
-  if (SCB_CCR_DC_Msk != (SCB_CCR_DC_Msk & SCB->CCR)) SCB_EnableDCache();
-#endif
-
+void board_init(void) {
+  BOARD_ConfigMPU();
```

Comment:
BOARD_ConfigMPU() is mcu-specific, this work with rt1170 but I have issue with rt1060 and other smaller variant such as 1015. I will comment this for now (in the next push/pr). we can generate and move these into its own board later when having time. :)

Reply:
yeah, I think mcxepersso pinconfig can generate mpu code (not sure), will do that later on. trying to add imxrt1064 to test both device and ehci mode and need to disable this to get it work.